lib/ansible/modules/cloud/azure/azure_rm_virtualnetworkpeering_facts.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Yunge Zhu <yungez@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_virtualnetworkpeering_facts
+version_added: "2.8"
+short_description: Get facts of Azure Virtual Network Peering.
+description:
+    - Get facts of Azure Virtual Network Peering.
+
+options:
+    resource_group:
+        description:
+            - Name of a resource group where the vnet exists.
+        required: True
+    name:
+        description:
+            - Name of the virtual network peering.
+    virtual_network:
+        description:
+            - Virtual network to be peered.
+            - It can be name of virtual network in same resource group.
+            - It can be virtual network resource id.
+            - It can be a dict of resource_group and name.
+        required: True
+
+extends_documentation_fragment:
+    - azure
+
+author:
+    - Yunge Zhu (@yungezz)
+'''
+
+EXAMPLES = '''
+    - name: Get virtual network peering by name
+      azure_rm_virtualnetworkpeering_facts:
+        resource_group: myResourceGroup
+        name: vnet_peer1
```

Comment:
i would move name param to the end, just for consistency.
ALso we could call it **myVnetPeering**